### PR TITLE
BACKLOG-16203: Fix glob-parent security vulnerabilities in sitemap.

### PR DESCRIPTION
**Pull request by XiangYuHe**
## JIRA

https://jira.jahia.org/browse/BACKLOG-16203

## Description

Update the glob-parent per the solution route.
The reason is our dependency is based on babel: https://github.com/babel/babel/blob/main/package.json
The babel dependencies haven't update yet.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -114,6 +115,7 @@
     "webpack-cli": "^4.2.0",
     "webpack-node-externals": "^1.7.2",
     "trim-newlines": "^3.0.1",
-    "css-what": "^5.0.1"
+    "css-what": "^5.0.1",
+    "glob-parent": "^5.1.2"
```
Comment (gflores-jahia) [inline package.json:119]: I'm not sure if it's required to add these as dev dependencies. If it works without it, we should remove.

**Review by gflores-jahia — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (XiangYuHe) [reply package.json:119]: Yup no problem once we confirm the convention we can delete the old obsolete code

**Comment by gflores-jahia** [comment]
Adding comment from @bpapez: 

> setting also the package designation in the resolutions element to a limited dependency path would also help to better understand why the resolution has been put there and to ease removing it

**Comment by XiangYuHe** [comment]
> Adding comment from @bpapez:
> 
> > setting also the package designation in the resolutions element to a limited dependency path would also help to better understand why the resolution has been put there and to ease removing it

Updated the possible dependencies with more specifications

File: package.json [hunk 2]
```diff
@@ -70,8 +70,10 @@
   "resolutions": {
     "graphql": "15.4.0",
     "@jahia/data-helper/graphql": "^15.4.0",
-    "trim-newlines": "3.0.1",
-    "css-what": "5.0.1"
+    "**/meow/trim-newlines": "3.0.1",
```
Comment (gflores-jahia) [inline package.json:73]: we should also use caret versions in resolutions

**Review by gflores-jahia — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (XiangYuHe) [reply package.json:73]: Sure I will update those ones

**Review by gflores-jahia — DISMISSED** [review]

**Review by gflores-jahia — APPROVED** [review]

**Pull request merged**